python/demo-braindead-trigger.sh

```diff
@@ -0,0 +1,12 @@
+#/bin/sh
+
+# This file is actually meant to be executed
+# Should fade out when you Ctrl+C
+
+while :
+do
+    oscsend localhost 5006 /input/left_nail
```

Comment:
As things are, it doesn't matter if we're receiving a 0 or a 1. Receiving an OSC message here means the trigger is active.